sdk/js/src/CacheStore.ts

```diff
@@ -17,13 +11,15 @@ export abstract class CacheStore {
         this.logger = logger
     }
 
-    private save(storeKey: string, data: unknown): void {
+    save(storeKey: string, data: unknown): void {
```

Comment:
separate save and load into their own `Storage` class and have this class take an instance of `Storage` in its constructor